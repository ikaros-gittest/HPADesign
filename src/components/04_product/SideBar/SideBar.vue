<template>
    <v-navigation-drawer color="primary" dark app>
        <v-list-item>
            <v-list-item-content>
                <v-list-item-title>
                    HPADesign
                </v-list-item-title>
                <v-list-item>
                    Concept
                </v-list-item>
                <v-list-item>
                    Airfoil
                </v-list-item>
                <v-list-item>
                    Aerodynamics
                </v-list-item>
                <v-list-item>
                    Structure
                </v-list-item>
            </v-list-item-content>
        </v-list-item>
    </v-navigation-drawer>
</template>