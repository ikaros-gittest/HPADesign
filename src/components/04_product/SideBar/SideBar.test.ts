import { mount } from '@vue/test-utils';
import SideBar from './SideBar.vue';

describe('SideBar', () => {
    it('snapshot test', () => {
        const wrapper = mount(SideBar);
        expect(wrapper.element).toMatchSnapshot();
    });
});