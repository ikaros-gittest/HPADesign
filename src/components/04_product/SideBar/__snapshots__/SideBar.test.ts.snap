// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar snapshot test 1`] = `
<v-navigation-drawer
  app=""
  color="primary"
  dark=""
>
  <v-list-item>
    <v-list-item-content>
      <v-list-item-title>
        
                HPADesign
            
      </v-list-item-title>
       
      <v-list-item>
        
                Concept
            
      </v-list-item>
       
      <v-list-item>
        
                Airfoil
            
      </v-list-item>
       
      <v-list-item>
        
                Aerodynamics
            
      </v-list-item>
       
      <v-list-item>
        
                Structure
            
      </v-list-item>
    </v-list-item-content>
  </v-list-item>
</v-navigation-drawer>
`;
